GoogleForCreators/web-stories-wp
# Templates: Update template using Andada

## Context



Acceptance Criteria can be found in original ticket. https://github.com/GoogleForCreators/web-stories-wp/issues/10829


## Summary

Replaces the Andada Font in the Food & Stuff template with Andada Pro.

Also adds a data migration to update existing stories automatically.


## Relevant Technical Choices



## To-do



## User-facing changes



## Testing Instructions






This PR can be tested by following these steps:

1. Create a new story from the Food & Stuff template and ensure that everything looks fine
2. Text elements should be using the Andada Pro font


## Reviews

### Does this PR have a security-related impact?



No

### Does this PR change what data or activity we track or use?



No

### Does this PR have a legal-related impact?



No

## Checklist




---



Fixes #10829
